packages/core/theme/src/components/input.ts

```diff
@@ -46,6 +46,7 @@ const input = tv({
       "data-[has-end-content=true]:pe-1.5",
       "file:cursor-pointer file:bg-transparent file:border-0",
       "autofill:bg-transparent bg-clip-text",
+      "input-search-cancel-button-none",
```

Comment:
i think we should hide it only when isClearable is true. Otherwise, it will break the existing behaviour for the false case.

Reply:
But lets say, isClearable is false. Then it will still have a clear button in chrome(when the type is search), which shouldn't be the case right ?
Also I tested in chrome, did you build it ? Because the changes are not getting reflected without build.

Reply:
- if `isClearable` is `false` and `type=search`, then it should show webkit-search-cancel-button because we don't want to break the native behaviour. 
- if `isClearable` is `true` and `type=search`, then it should only show NextUI Close button (since it supports onClear handle) and hide webkit-search-cancel-button one.
- True. The theme package needs to be built.